doc/nrf/libraries/others/audio_module.rst

```diff
@@ -0,0 +1,120 @@
+.. _lib_audio_module:
+
+Audio module
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The audio module library is an interface to audio processing functions that coordinate audio states and the exchange of audio-related data of an LE Audio application, such as :ref:`nrf53_audio_app`.
```

Comment:
Is it LE Audio or Bluetooth LE Audio that is correct? @greg-fer

Reply:
I guess both: https://www.bluetooth.com/learn-about-bluetooth/feature-enhancements/le-audio/